### ISSUE TITLE
False-positive message with swapchan, but without WSI

### ISSUE THREAD

**Issue by Neko-tech-priest** [KhronosGroup/Vulkan-ValidationLayers#11477]
**Environment**
 - OS: Linux
 - GPU and driver version: RX 6600XT, Mesa 25.3.3
 - Options enabled: Sumbit time validation, AMD-specific, Performance

**Describe the Issue**
Without VK_KHR_swapchain validation layer don't know when frame changed and thinking what i bind the same pipeline again, but i have `vkEndCommandBuffer` and `vkQueueSubmit2` every frame with `vkWaitForFences` waiting, with WSI implementation was fine

**Expected behavior**
Nothing messages

Instead, i have message:
```sh
Validation Performance Warning: [ BestPractices-Pipeline-SortAndBind ] | MessageID = 0x6d0c146d
vkCmdBindPipeline(): [AMD] [NVIDIA] Pipeline VkPipeline 0x380000000038 was bound twice in the frame. Keep pipeline state changes to a minimum, for example, by sorting draw calls by pipeline.

```
**Additional context**
I make pure Wayland application, without libwayland-client

**Comment by spencer-lunarg** [comment]
So looking at the Best Practice code, the logic is simply

- at `vkCmdBindPipeline` check if same pipeline was bound twice (if not add)
- at `vkQueuePresentKHR` reset 

So I just realized looking at this, this is not even checking if graphics

@Neko-tech-priest is your Pipeline a compute pipeline?

**Comment by Neko-tech-priest** [comment]
@spencer-lunarg No, it's graphics pipeline
As i said, `vkCmdBindPipeline` bind only once(even in render frame it's called only 1 time, no other pipelines) and `vkQueuePresentKHR` require `VK_KHR_swapchain`, what impossible use without WSI
